l10n/de_DE/LC_MESSAGES/remoteevent.po

```diff
@@ -1349,8 +1418,70 @@ msgstr ""
 
 #: templates/CRM/Remoteevent/Form/RegistrationConfig.hlp
 msgid ""
-"If you tick this box, registrations will be assigned the \"Awaiting approval"
-"\" status and you will need to manually review and approve each "
+"Whether participants can register with themselves up to 9 other participants "
+"using the profile to be selected below."
+msgstr ""
+"Bestimmt, ob Veranstaltungsteilnehmer mit ihrer Anmeldung bis zu 9 weitere "
+"Teilnehmer anmelden können, unter Verwendung des unterhalb ausgewählten "
+"Profils."
+
+#: templates/CRM/Remoteevent/Form/RegistrationConfig.hlp
+msgid "The registration profile to use for additional participants."
+msgstr "Zur Anmeldung zusätzlicher Teilnehmer verwendetes Anmeldeprofil."
+
+#: templates/CRM/Remoteevent/Form/RegistrationConfig.hlp
+msgid ""
+"Limit the number of additional participants that can be registered in a "
+"single booking."
+msgstr ""
+"Beschränkung der Anzahl zusätzlicher Teilnehmer, die in einer "
+"Einzelanmeldung mit angemeldet werden können."
+
+#: templates/CRM/Remoteevent/Form/RegistrationConfig.hlp
+msgid ""
+"Eg: if you choose '2' then the lead booker can bring 2 guests; there would "
+"be a limit of 3 participants in total per booking."
+msgstr ""
+"Wenn Sie z. B. '2' auswählen, kann der Teilnehmer bis zu 2 Gäste mitbringen; "
```

Comment:
I think at least in German there should be used double quotes instead of single quotes for `2`.

Reply:
Agreed - I think this is a Core string though, so would actually not need a translation at all; not sure how to mark those though. Fixed in the PO anyway.